autoload/smoothie.vim

```diff
@@ -1,3 +1,16 @@
+if exists('g:loaded_smoothie')
+  finish
+endif
+let g:loaded_smoothie = 1
```

Comment:
Why is this needed? I thought `autoload` scripts are sourced only once by Vim anyway?